### ISSUE TITLE
Implement new dashboard layout

### ISSUE THREAD

**Issue by hellais** [ooni/probe#1094]
This issue is about implementing https://sketch.cloud/s/8vqJq/v/g119vz/ inside of probe-mobile.

**Comment by hellais** [comment]
This is duplicate of #1039

**Comment by lorenzoPrimi** [comment]
Reopening. This is about implementing the screens in the #870 card

**Comment by lorenzoPrimi** [comment]
Regarding the dashboard:
- Are title and subtitle black or some gray?
- Do I need new icons or I just fill the ones I have with the default test color? (they don't seem to be of a unique color)
- What is the height of these rows. And size of the icon?
- What is the "last tested 1 day ago" label about? the last test ran of any type?

**Comment by hellais** [comment]
> Regarding the dashboard:
> * Are title and subtitle black or some gray?

They are black.

You can use the sketch inspect to see the color picker: https://sketch.cloud/s/8vqJq/v/g119vz/a/g2YPPq/r/AZP8MY#Inspector.

> * Do I need new icons or I just fill the ones I have with the default test color? (they don't seem to be of a unique color)

The icons are custom and have a gradient on them. How would you like me to export them as?

> * What is the height of these rows. And size of the icon?

Check the sketch inspector: https://sketch.cloud/s/8vqJq/v/g119vz/a/g2YPPq/r/AZP8MY#Inspector.

> * What is the "last tested 1 day ago" label about? the last test ran of any type?

It is the last time the "run all tests" button was triggered.

> * I need that OONI Probe logo. I only have the "OONI" part.

If you tell me in what format you need it I can export it.

> * Why are we switching from a centered logo to a left aligned one?

I find it to be a bit more balanced this way. I was assuming we had the settings in the top right conerner, but I think even without that it's still more balanced to have it left aligned.

**Comment by lorenzoPrimi** [comment]
> The icons are custom and have a gradient on them. How would you like me to export them as?
> 
I need `pdf` for iOS and `svg` (that I will convert to `xml`) for Android. 
In Adobe XD I had the export option for both formats.

> > * What is the "last tested 1 day ago" label about? the last test ran of any type?
> 
> It is the last time the "run all tests" button was triggered.

What if the user has ran a single "Websites" test?
(at the moment we save only the last run for every single test)

> > * I need that OONI Probe logo. I only have the "OONI" part.
> 
> If you tell me in what format you need it I can export it.

Same as above

> > * Why are we switching from a centered logo to a left aligned one?
> 
> I find it to be a bit more balanced this way. I was assuming we had the settings in the top right conerner, but I think even without that it's still more balanced to have it left aligned.

In iOS I think we need to keep the centered way, in Android is doable.

**Comment by hellais** [comment]
> I need pdf for iOS and svg (that I will convert to xml) for Android.
> In Adobe XD I had the export option for both formats.

Ok. I don't use XD, but I can export it in those formats easily.

> What if the user has ran a single "Websites" test?

I think we should not populate the last run label in that case.

> In iOS I think we need to keep the centered way, in Android is doable.

It would be better if we could left align it as in the mock on iOS too. That said iOS is a secondary platform so it's not really that important.

Please start the implementation of this feature on Android and iOS only after.

**Comment by hellais** [comment]
>What if the user has ran a single "Websites" test?
> I think we should not populate the last run label in that case.

If it is too complicated to record when the "Run all" was last triggered, we can also just show the last runtime of any test, even if it's not triggered via "Run all", but also just an individual card.

**Comment by lorenzoPrimi** [comment]
iOS Testflight has been released.
Android apk to test is here: https://drive.google.com/open?id=1YH4E_lPe-FCLAIMPCY2CcJIYViaEuKR5
